### ISSUE TITLE
Settings should be named in the title bar Windows 

### ISSUE THREAD

**Issue by thoeg** [edrlab/thorium-reader#2556]
I am familiarizing  myself with Thorium iu using JAWS. Learning the diffrent keyboard shortcuts.
They are hard to get but otherwise seem to work, however it would make the app more friendly and easier to understand if different windows hwere indicated in the title bar.
So settings when opened  opens ok but the title bar is the same as the book or place  in Thorium cyou came from.
If Settings was the title of the Window it would make navigation and orientation much  easier for a screen reader user. This may also be the case in other places in the app  I am not  sure.
Win11 2023h2
jaws 2024.

**Comment by panaC** [comment]
Hello, sorry but I didn't quite understand what you said, can you explain me what "settings" is for you?

**Comment by thoeg** [comment]
Hi, The settings I refer to is the app settings. As a new user you would have to go there often to read the list of assigned shortcuts. This list is by the way also hard to read with a screen reader, and the list should be made more accessible and added to the documentation. In this way users can read it like other information and copy it to  our own documents or even hardware like a braille notetaker.  But as it is now you will often have to go into ssettings to check all the different shortcut keys.Here you can easily lose the overview of where you are. Either in the  settings dialog or in the book you came from.This could be made more obvious if the title of the  window with settings included the word setting. As it works now nothing changes when you open or  close settings. This is not a problem for users that can see the entire screen, but it can be a problem for screen reader  users that works with a much smaller part of the screen basically what is in focus at the time. Thorium - Thorium Reader Documentation EnglishReading with a screen reader Reading the text now If I open settings press ctrl+s I hear: Thorium - Thorium Reader Documentation English dialoggruppeCustomize text formatting check boks ikke markeret So I get the new focus. But let us say i get distracted by something comes back to the book Iand wwant to check where I am. Now reading the title foof the windo in Jaws jawsKey+t Now I hear: Thorium - Thorium Reader Documentation English So what I am told is the name of the book I was reading before I opened settings. This in my view is a bug since I am not in the book but in settings. This can be  obvious because jaws sometiems will add extra information about the current heading, but the first infromation you get is in my view misleading. Regards Claus  Sendt fra Mail til Windows Fra: Pierre LerouxSendt: 1. oktober 2024 14:57Til: edrlab/thorium-readerCc: thoeg; AuthorEmne: Re: [edrlab/thorium-reader] Settings should be named in the title bar Windows (Issue #2556) Hello, sorry but I didn't quite understand what you said, can you explain me what "settings" is for you?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by gautierchomel** [comment]
@thoeg The shortcut list is available online from the support and resources page [Keyboard shortcuts](https://thorium.edrlab.org/en/th3/400_ressources/402_keyboard-shortcuts/). Please remember that this page is to be updated soon and that this website is still missing a full accessibility assessment. If you meet any blocker, please report it to the dedicated [thorium doc issue tracker.](https://github.com/edrlab/thorium-reader-doc/issues/new) 

I notice a confusion between the Application Settings panel, available from the main Thorium window where the libraries stand, and the User Preference panel available from each book window. The second one does not give access to shortcuts.

Both are not windows, but Panels. That's the reason why Jaws contextualise it in the _Thorium - Thorium Reader Documentation_ window.

**Comment by gautierchomel** [comment]
@panaC a way could be to add a H* title to those panels so one can figure out where he is.
